### DISCUSSION TITLE
Cant use expander and custom sort on same column

### DISCUSSION THREAD

**Discussion by morea030** [TanStack/table#1312] [General]
# Using v6?
yes

**Describe the bug**

Here is the short replication
https://codesandbox.io/embed/2v4rln937r

Im trying to make custom sort method for the "Monthly Renatal" column.

 If the expander setting is activated (line 68 in the example) when I try to sort that column the sortMethod(a,b) will be called with undefined values (which can be confirmed with the console log on the line 71). 

If on the other hand expander setting is commented, the sortMethod will be called with proper values.

I think this is a bug, but if Im doing something wrong, please let me know

**Comment by morea030** [comment]
Also if there is another way of achieving what Im trying to accomplish here, please let me know

**Comment by tannerlinsley** [comment]
Please ask this question on our support forum: https://spectrum.chat/react-table

**Comment by dreamz89** [comment]
As the spectrum chat is now closed and the issue never got replies, may I reopen this issue?
I also have a problem with sorting an expandable column, for task ID numbers

**Comment by tannerlinsley** [comment]
I’m assuming this is for v6?

**Comment by dreamz89** [comment]
> I’m assuming this is for v6?

yes

**Comment by tannerlinsley** [comment]
A discussion should suffice then :)

**Comment by dmitriigaidarji** [comment]
@dreamz89 you can fix it by removing one line in src/methods.js like this: 
https://github.com/dmitriigaidarji/react-table/commit/65332b56dc97a30f2da41e9a5fdf8443998ab8f6
